instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqTracingBuilder.java

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
For `setPropagationEnabled`

`Sets whether the trace context should be written from producers / read from consumers for propagating through messaging.`

Reply:
This is currently the case, written it in TracingSendMessageHookImpl, and read it in RocketMqConsumerTracer.

Reply:
Sorry I meant you can copy-paste that as the javadoc for the `setPropagationEnabled` method which currently doesn't have javadoc (and GitHub didn't let me comment on it previously since I was looking at the commit, not PR, oops.

Reply:
That's OK. :smile: